python/lsst/obs/huntsman/ingest.py

```diff
@@ -79,8 +78,8 @@ def translate_visit(self, md):
         """
         date_obs = md['DATE-OBS']  # This is a string
         datestr = ''.join([s for s in date_obs if s.isdigit()])
-        assert len(datestr) > 17, "Date string too long for int64."
-        return datestr
+        assert len(datestr) > 17, "Date string too long."
```

Comment:
I don't think this is doing what you're intending? `assert 17>17` would evaluate to false here and trigger the error